c2corg_ui/templates/waypoint/edit.html

```diff
@@ -735,14 +735,22 @@ <h2 class="heading show-phone" translate>description</h2>
 
             ## DESCRIPTION
             <div id="description-group" class="form-group data full">
-              <label translate>description</label>
-              <textarea name="description" ng-model="waypoint.locales[0].description" class="form-control description" placeholder="{{'Describe here the waypoint' | translate}}"></textarea>
+              <label translate ng-if="type == 'access'">road access</label>
+              <label translate ng-if="type != 'access'">description</label>
+              <textarea name="description" ng-model="waypoint.locales[0].description" class="form-control description"
+                placeholder="{{ type =='access' ? mainCtrl.translate('Describe here the waypoint') : mainCtrl.translate('Describe road access') | translate}}"></textarea>
             </div>
 
             ## ACCESS
-            <div class="form-group data full" ng-if="['access', 'local_product', 'climbing_outdoor'].indexOf(type) > -1">
-              <label translate>access</label>
-              <textarea name="access" ng-model="waypoint.locales[0].access" class="form-control" placeholder="{{'Describe access' | translate}}"></textarea>
+            <div class="form-group data full" ng-if="['access', 'local_product', 'climbing_outdoor', 'climbing_indoor', 'hut'].indexOf(type) > -1">
+              <label translate ng-if="type == 'access'">pt access</label>
```

Comment:
pt access?
What does this mean? Are you sure translators will understand how to translate this in Transifex?